source/scripts/router.js

```diff
@@ -46,4 +49,29 @@ export class Router {
       .querySelector("notification-select-cookbook")
       .classList.add("hidden");
   }
+
+  /**
+   * This function navigates to the previous page by toggling the "hidden" name
+   * on the element's classList. Only used by the back button.
+   */
+  return() {
+    //Get references to elements
+    let pageElement = document.querySelector(this.previousPage);
+    let currentElement = document.querySelector(this.currentPage);
+
+    //Hide the current page
+    currentElement.classList.toggle("hidden");
+    if (pageElement.classList.contains("hidden")) {
+      //store the previousPage int the temp variable
+      let temp = this.previousPage;
+      //set previous page to the current page
+      this.previousPage = this.currentPage;
+      //Set new current page to page
+      this.currentPage = temp;
+      //Toggle hidden on page
+      pageElement.classList.toggle("hidden");
+      //Scroll to top of page
+      document.querySelector("html").scrollTop = 0;
+    }
+  }
```

Comment:
You actually don't need this function as it performs the same exact function as `navigate`. Instead, we could use a getter to get the `previousPage` in order to navigate to it
```suggestion

  get prevPage() {
    return this.previousPage;
  }
```